cmake/std/atdm/cee-rhel6/environment.sh

```diff
@@ -110,6 +110,9 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "GNU-4.9.3_OPENMPI-1.10.2" ]] ; then
   export ATDM_CONFIG_SUPERLUDIST_LIBS=${SUPERLUDIST_ROOT}/lib/libsuperlu_dist_4.2.a
 elif [ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.2_MPICH2-3.2" ]; then
   module load sparc-dev/intel-18.0.2_mpich2-3.2
+  export OMP_NUM_THREADS=3 # Because Si H. requested this
```

Comment:
@nmhamster and @rppawlo, see how it is changed to 3.

Reply:
For reference, `module load sparc-dev/intel-18.0.2_mpich2-3.2` has been setting `OMP_NUM_THREADS=1` so AFAIK this build has not tested more than one OpenMP thread in a long time.